src/matrix-utils.js

```diff
@@ -25,12 +28,69 @@ function waitForSync(client) {
   });
 }
 
+function addScript(src) {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement("script");
+    script.setAttribute("src", src);
+    script.onload = resolve;
+    script.onerror = reject;
+    document.body.appendChild(script);
+  });
+}
+
 export async function initClient(clientOptions) {
+  await addScript(olmJsPath);
+  await window.Olm.init({ locateFile: () => olmWasmPath });
+
+  let indexedDB;
+
+  try {
+    indexedDB = window.indexedDB;
+  } catch (e) {}
+
+  const storeOpts = {};
+
+  if (indexedDB && localStorage && !import.meta.env.DEV) {
```

Comment:
Fwiw I think the bit you need is the crypto store below - this shouldn't really be necessary to get crypto working (although it will speed up your subsequent loads).

Reply:
Yeah Matthew and I discussed adding this to Element Call so I figured it was good to do it here in addition to the crypto store.